rhino/README.md

```diff
@@ -0,0 +1,112 @@
+# Introduction
+
+Rhino supports two editors: IronPython and CPython. The following tutorials explain how to install and use compas on both.
+
+# Installation - Editor: IronPython
+
+- **Step 1:** Install Anaconda: https://www.anaconda.com/products/distribution
+
+- **Step 2:** Anaconda Prompt commands:
+
+    ```bash
+
+    conda create -n compas-dev -c conda-forge compas
+    conda activate compas-dev
+    python -m compas_rhino.install -v7.0
+    ```
+
+    ![Conda Environment](conda_environment.png)
```

Comment:
The current recommendation is not to use admin mode to install compas

Reply:
Removed administration suggestion.